scripts/spack/configs/blueos_3_ppc64le_ib_p9/packages.yaml

```diff
@@ -185,9 +185,9 @@ packages:
   hdf5:
     variants: ~shared~mpi
   hypre:
-    version: [2.20.0]
+    version: [2.24.0]
```

Comment:
Did you verify that spack actually used this version? because listing versions here is a suggestion and it may decide to flip it.

Reply:
I didn't but I'll check now.

(note though that Axom doesn't directly use hypre -- it is only used indirectly through mfem)